backend/app/Models/Tenant.php

```diff
@@ -22,6 +22,17 @@ class Tenant extends Model
         'deleted_at' => 'datetime',
     ];
 
+    public function currentSubscriptionPlan()
```

Comment:
細かいところで恐縮ですが

```suggestion
    public function currentSubscriptionPlan(): void
```

その他、型をつけれるところはつけたほうがいいっすね